runelite-client/src/main/java/net/runelite/client/plugins/eeng1n/woodcutting/threetickteaks/ThreeTickTeaksConfig.java

```diff
@@ -0,0 +1,26 @@
+package net.runelite.client.plugins.eeng1n.woodcutting.threetickteaks;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("threeTickTeaks")
+public interface ThreeTickTeaksConfig extends Config {
+    @ConfigItem(
+            keyName = "guide",
+            name = "How to use",
+            description = "How to use this plugin",
+            position = 0
+    )
+    default String GUIDE() {
+        return "1. Have your inventory setup for 3T Teaks \n2. Enable Plugin";
```

Comment:
It's always nice to sum up the item requirements and stat requirements and location requirement for the script